e2e-tests/trailing-slash/cypress/integration/static.js

```diff
@@ -0,0 +1,40 @@
+import { assetPageVisits } from "../support/utils/trailing-slash"
+
+const IS_BUILD = Cypress.env(`IS_BUILD`)
+
+describe(`static directory`, () => {
+  describe(`static/something.html`, () => {
+    if (IS_BUILD) {
```

Comment:
From @j0sh77:

A minor thing - fom a reporting standpoint, could we do something like:
```
const conditionalIf = (condition) => condition ? it : it.skip;
```
And then:
```
conditionalIf(IS_BUILD)(() => {...});
```
That would help our reporting and potentially be less confusing when reading through cypress output